Dockerfile.notebook

```diff
@@ -0,0 +1,21 @@
+FROM nvcr.io/nvidia/pytorch:20.03-py3
+
+RUN apt update && apt install curl gnupg
+RUN curl https://bazel.build/bazel-release.pub.gpg | apt-key add -
+RUN echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list
+
+RUN apt update && apt install bazel-3.2.0
+RUN ln -s /usr/bin/bazel-3.2.0 /usr/bin/bazel
+
+RUN pip install torch==1.5.0
+
+COPY . /workspace/TRTorch
+COPY ./notebooks/WORKSPACE  /workspace/TRTorch
+
+WORKDIR /workspace/TRTorch
+RUN bazel build //:libtrtorch --compilation_mode opt
```

Comment:
Do we want to build both the C++ and Python libraries? or do we just need python?

Reply:
in this particular example I guess just Python? But we might have more examples with C++ lib soon I reckon.